src/datatype.c

```diff
@@ -523,6 +525,7 @@ void jl_compute_field_offsets(jl_datatype_t *st)
         //else if (st->layout->fielddesc_type != 0) // GC only implements support for this
         //    isinlinealloc = 0;
         isinlinealloc = 0;
+        isbitstype = 0;
```

Comment:
we just asserted this was zero on the previous line

Reply:
Yes I have a fix for this.

Reply:
It seemed like there might be a case above where we did set not-inlinealloc (from references-name), but don’t simultaneously also set not-isbitstype